readthedocs/doc_builder/environments.py

```diff
@@ -539,6 +542,8 @@ def create_container(self):
                     }
                 }),
                 detach=True,
+                environment = {'READTHEDOCS_VERSION' : self.version.slug,
```

Comment:
This isn't needed. The top part is the only bit that will run in prod.

Remove these lines, and I'll merge.